src/characters/universal-bob.ts

```diff
@@ -172,4 +173,22 @@ export class tDecDecrypter {
     const config = JSON.parse(json, base64ToU8Receiver);
     return tDecDecrypter.fromObj(config);
   }
+
+  public equals(other: tDecDecrypter): boolean {
```

Comment:
Some implementations of `equals` will remain hand-rolled until https://github.com/nucypher/nucypher-core/issues/56 is resolved